### ISSUE TITLE
initial intigration

### ISSUE THREAD

**Issue by wwlwpd** [jasonfleming/asgs#206]
Add initial calls to ASGS for registration with the service in a generica and extensible way.

**Comment by jasonfleming** [comment]
@wwlwpd I'm hoping that the `asgs.instance.status.json`, the `hook.status.json` and the `scenario.status.json` etc posted to the OPeNDAP server will suffice for this ... what do you think?

**Comment by wwlwpd** [comment]
Yeah this is a very poorly worded issue, I have no idea what I was intending here. Closing 😄
